### PR TITLE
Add tooltip to outdated notice in mentoring session

### LINKED ISSUE #5908: Tooltip to explain outdated
Wherever the Outdated notice appears, it should be accompanied by a tooltip explaining what this means.

image

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/exercism/issues/5908.

### For mentors
Screen Shot 2021-09-29 at 4 32 33 PM

### For students
Screen Shot 2021-09-29 at 4 33 02 PM

### CODE REVIEW

**Comment by iHiD** [comment]
Red @kntsoriano.

**Comment by iHiD** [comment]
(Looks good though!)

File: test/system/components/student/mentoring_session/outdated_solution_test.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+require_relative "../../../../support/capybara_helpers"
+
+module Components
+  module Student
+    module MentoringSessionTest
```
Comment (iHiD) [inline test/system/components/student/mentoring_session/outdated_solution_test.rb]:
```suggestion
    module MentoringSession
```

**Comment by iHiD** [comment]
Great work :)

**Comment by kntsoriano** [comment]
Thanks man :) I've learned a great deal here on passing a component rather than a boolean that says whether to display a certain component or not. It's like dependency injection in React.

**Comment by iHiD** [comment]
That's cool :)

**Pull request merged**